### DISCUSSION TITLE
Oculus Quest

### DISCUSSION THREAD

**Discussion by VitArts** [xtrime-ru/antizapret-vpn-docker#11] [Q&A]
Привет подскажи пожалуйста в чем может быть проблема -  не работают игры на oculus quest при загрузке просто зависает лоадер у любой игры, если отключить VPN (либо просто включить дефолтный vpn от antizapret) то все становится норм.

Пробывал удалить все домены из custom листа, проблема остается, так же добавлял туда домены, если добавить oculus.com то сервис вообще отваливается, сайт не открывается, в консоли у домена oculus сразу ошибки CORS  и др.

Делал tcpdump, прописывал все основные домены из него (кроме meta и oculus т.к если их прописать то отваливается вообще все).

Что можно еще сделать, как победить?

**Comment by xtrime-ru** [comment]
Привет! У некоторых сервисов запросы, которые пришли в неправильный  регион просто откидываются. Так происходит из за того, что DNS на удаленном сервере резолвит IP в одном регионе, а идешь ты по нему из другого. 
Ранее я с подобным сталкивался у Apple. Добавь сюда домены, которые надо резолвить через DNS яндекса (не забудь добавить точки после доменов): https://github.com/xtrime-ru/antizapret-vpn-docker/blob/master/patches/kresd.conf#L9 и сделай 

```
docker compose build
docker compose up -d
```

Если проблема решится - то скинь скинь список доменов, что бы я их добавил в репозиторий.

**Reply by VitArts** [reply]
Хорошо, спасибо. Буду дома, тестану.
